sdk/include/mcm_dp.h

```diff
@@ -207,6 +207,7 @@ typedef struct {
 /* rdma format */
 typedef struct {
     size_t transfer_size;
+    int queue_size;
```

Comment:
Does this change affect the SDK API declared in `mesh_dp.h`?

Reply:
mcm_rdma_args is not used inside mesh_dp.h